packages/vuetify/src/presets/default/index.ts

```diff
@@ -0,0 +1,67 @@
+// Styles
+import '../../styles/main.sass'
+
+// Locale
+import { en } from '../../locale'
+
+// Icons
+// TODO: Enable for v3
+// import mdi from '../../services/icons/presets/mdi'
+
+// Types
+import { VuetifyPreset } from 'vuetify/types/services/presets'
+
+export const Preset: VuetifyPreset = {
```

Comment:
I'd rather export it lowercased. "Preset" looks more like a class name, also it's more convenient to type `preset,` instead of `preset: Preset,`